src/routes/docs/tutorials/pagination/step-5/+page.markdoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: tutorial
+title: Environment setup
+description: Setup the environment for seeding the database
+step: 5
+---
+
+# Seeding the collection {% #seeding-the-collection %}
+We'll now create a script that populates documents from a `setup.json` file.Create a new file `./db/setup.js` in the root of your project directory for this purpose.
+
+Insert the following code, make sure to replace `[YOUR_DATABASE_ID]` and `[YOUR_COLLECTION_ID]` with your respective project and collection IDs.
+
+```js
+import { databases } from "../src/lib/appwrite.js";
+import { ID } from "appwrite";
+
+export const TODOS_DATABASE_ID = "[YOUR_DATABASE_ID]";
+export const TODOS_COLLECTION_ID = "[YOUR_COLLECTION_ID]";
+const dataURL = "https://jsonplaceholder.typicode.com/users/1/todos";
+
+async function seedCollection() {
+  const response = await fetch(dataURL);
+  const data = await response.json();
+  console.log("Starting to seed collection...");
+  try {
+    for (let i = 0; i < data.length; i++) {
+      const item = data[i];
+      let promise = await databases.createDocument(
+        TODOS_DATABASE_ID,
+        TODOS_COLLECTION_ID,
```

Comment:
since the app is created using Vite, these env variables should be like this: 
```suggestion
        import.meta.env.APPWRITE_DATABASE_ID,
        import.meta.env.APPWRITE_COLLECTION_ID,
```
we can tell them to write them like this so the env variables will not be exposed to the client.